src/webp_converter.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const imagemin = require('imagemin');
+const imageminWebp = require('imagemin-webp');
+const path = require('path');
+
+
+// Recursively converts all jpg and png files to webp in docs directory tree
+(async () => {
+    async function convertImagesToWebp(dirPath) {
+        // Converts all jpg and png files in current directory to webp
+        await imagemin([`${dirPath}/*.{jpg,png}`], {
+            destination: dirPath,
+            plugins: [
+                imageminWebp({ quality: 65 }),
+            ],
+        });
+
+        const files = fs.readdirSync(dirPath);
```

Comment:
nitpick - you have async function so it's always better to use async `fs.readdir` rather than blocking `readdirSync`:

`const { readdir } = require('fs/promises')`

`... await readdir`.

It's not a big deal here but to keep everything clean